packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -158,24 +159,23 @@ class FlutterPlugin implements Plugin<Project> {
             }
         }
 
-        if (shouldShrinkResources(project)) {
-            String flutterProguardRules = Paths.get(flutterRoot.absolutePath, "packages", "flutter_tools",
-                    "gradle", "flutter_proguard_rules.pro")
-            project.android.buildTypes {
-                release {
-                    // Enables code shrinking, obfuscation, and optimization for only
-                    // your project's release build type.
-                    minifyEnabled true
-                    // Enables resource shrinking, which is performed by the
-                    // Android Gradle plugin.
-                    // NOTE: The resource shrinker can't be used for libraries.
-                    shrinkResources isBuiltAsApp(project)
-                    // Fallback to `android/app/proguard-rules.pro`.
-                    // This way, custom Proguard rules can be configured as needed.
-                    proguardFiles project.android.getDefaultProguardFile("proguard-android.txt"), flutterProguardRules, "proguard-rules.pro"
-                }
+        String flutterProguardRules = Paths.get(flutterRoot.absolutePath, "packages", "flutter_tools",
+                "gradle", "flutter_proguard_rules.pro")
+        project.android.buildTypes {
+            release {
```

Comment:
levers?

Reply:
when users create custom types, we should document what the right approach is to have the correct default settings associated. e.g. recommend

```groovy
production {
     initWith release         
}
```

rather than making one from scratch

Reply:
I don't think we are doing anything special, other than profile.

`release` is an official build type provided by the Android plugin. I think the Flutter docs can link to https://developer.android.com/studio/build/build-variants.